include/functions.h

```diff
@@ -1512,22 +1512,22 @@ void PreRender_SetValuesSave(PreRender* this, u32 width, u32 height, void* fbuf,
 void PreRender_Init(PreRender* this);
 void PreRender_SetValues(PreRender* this, u32 width, u32 height, void* fbuf, void* zbuf);
 void PreRender_Destroy(PreRender* this);
-void func_800C0F28(PreRender* this, Gfx** gfxp, void* buf, void* bufSave);
-void func_800C1258(PreRender* this, Gfx** gfxp);
-void func_800C170C(PreRender* this, Gfx** gfxp, void* fbuf, void* fbufSave, u32 r, u32 g, u32 b, u32 a);
+void PreRender_CopyImage(PreRender* this, Gfx** gfxp, void* img, void* imgDst);
+void PreRender_CopyImageRegionImpl(PreRender* this, Gfx** gfxp);
+void func_800C170C(PreRender* this, Gfx** gfxp, void* buf, void* bufSave, u32 r, u32 g, u32 b, u32 a);
 void func_800C1AE8(PreRender* this, Gfx** gfxp, void* fbuf, void* fbufSave);
-void func_800C1B24(PreRender* this, Gfx** gfxp, void* fbuf, void* cvgSave);
-void func_800C1E9C(PreRender* this, Gfx** gfxp);
-void func_800C1F20(PreRender* this, Gfx** gfxp);
-void func_800C1FA4(PreRender* this, Gfx** gfxp);
-void func_800C20B4(PreRender* this, Gfx** gfxp);
-void func_800C2118(PreRender* this, Gfx** gfxp);
+void PreRender_DrawCoverage8b(PreRender* this, Gfx** gfxp, void* img, void* cvgDst);
```

Comment:
I can't find this function anywhere in the C file, is it supposed to be here?
Same comment for `PreRender_DrawCoverage16b` and possible others

Also feel free to remove prototypes from this file if they are internal and not meant to be called externally, which I assume is the case for most functions in this system